liquibase-standard/src/main/java/liquibase/resource/Resource.java

```diff
@@ -29,6 +29,18 @@ public interface Resource {
      */
     boolean isWritable();
 
+    /**
+     *
+     * Default implementation
+     *
+     * @param   possibleFolder    Path to check for a folder
+     * @return  boolean
+     *
+     */
+    default boolean isFolder(String possibleFolder) {
+       return false;
+    }
```

Comment:
Since this effectively has no implementation for anything other than S3, isn't this a bit of a concern? Someone might use this method thinking that it will actually tell them if something is a folder, but it will only work for S3.